src/core/base-runner.js

```diff
@@ -27,8 +27,8 @@ function toRunnable(plug) {
         performance.mark(name + "-start");
       }
       try {
-        // Modern plugins are async or normal functions, take one argument (conf)
-        if (plug.run.length === 1) {
+        // Modern plugins are async or normal functions, take zero or one argument (conf)
+        if (plug.run.length <= 1) {
```

Comment:
nit: `<==` ... just as matter of style, we use strict equality throughout the project

Reply:
I think `<==` is not even a JS operator...

Reply:
Argh... I quit.

Reply:
I haven't heard there is a strict comparison operator in JavaScript, we probably should just ignore the type strictness here.

Reply:
You are too kind @saschanaz. I just need more coffee.